src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/AddArtworksModalContent.tsx

```diff
@@ -51,14 +60,33 @@ export const AddArtworksModalContent: FC<AddArtworksModalContentProps> = ({
 
   const handleSortChange = (option: string) => {
     setSort(option)
+
     setIsLoading(true)
 
-    relay.refetch({ sort: option }, null, error => {
-      if (error) {
-        console.error(error)
+    relay.refetchConnection(
+      ARTWORKS_PER_SCROLL,
+      err => {
+        console.error(err)
+      },
+      { sort: option }
+    )
+
+    setIsLoading(false)
```

Comment:
Surrounding `refetchConnection` with `setIsLoading` doesn't makes sense, unfortunately. `refetchConnection` finishes its execution immediately, and `isLoading` becomes `false` right away. That's why current behaviour of changing a sort option looks broken. When I change sort option, placeholder component is not displayed. I am looking for a solution, but let's not block this PR because of it

cc @dimatretyak

Reply:
you need to call `setIsLoading(false)` in `refetchConnection` callback. something like this

```javascript
relay.refetchConnection(
  ARTWORKS_PER_SCROLL,
  err => {
    if (err) {
      console.error(err)
    }

    setIsLoading(false)
  },
  { sort: option }
)
```